graph/State-Associated-Subsets.md

```diff
@@ -0,0 +1,113 @@
+# Pattern Name
+
+Microsoft Graph API Design Pattern
+
+
+
+Modeling state associated to a collection that may include all instances, an included subset, an excluded subset, no instances, or any combinations of the above.
```

Comment:
This pattern doesn't seem to be associating any state to a collection.  It feels more like defining a set of well known predicates for the purpose of determining subsets of a collection.   The state is the predicate attached to the policy (or whatever) that wants to operate on the collection.